### ISSUE TITLE
RTV6: Driver does not support the 0x46a6 PCI ID

### ISSUE THREAD

**Issue by yavuz650** [ubc-aamodt-group/vulkan-sim#11]
I installed vulkan-sim in a docker container along with dependencies. When I run the ray tracing app, the integrated Intel GPU is picked as the physical device, however, I get the following error.

```
Driver does not support the 0x46a6 PCI ID.
MESA-INTEL: error: ../src/intel/vulkan/anv_device.c:324: VK_ERROR_INCOMPATIBLE_DRIVER
FATAL: failed to enumerate (ERROR_INITIALIZATION_FAILED)

 0# 0x00005598A9C3FD76 in ./RayTracer
 1# 0x00005598A9C4184F in ./RayTracer
 2# 0x00005598A9C8592B in ./RayTracer
 3# 0x00005598A9C8647C in ./RayTracer
 4# 0x00005598A9C7C7C4 in ./RayTracer
 5# 0x00005598A9C8B3C0 in ./RayTracer
 6# 0x00005598A9C9C8F0 in ./RayTracer
 7# 0x00005598A9C495E6 in ./RayTracer
 8# __libc_start_main in /usr/lib/x86_64-linux-gnu/libc.so.6
 9# 0x00005598A9C4ABDE in ./RayTracer

```
Following is the ICD file

```
{
    "ICD": {
        "api_version": "1.2.145",
        "library_path": "/home/vulkan-sim-root/mesa-vulkan-sim/lib/lib/x86_64-linux-gnu/libvulkan_intel.so"
    },
    "file_format_version": "1.0.0"
}
```

I suspect something went wrong during mesa build. Any hints?

I'm running vulkan-sim in a Ubuntu 20.04 container with Cuda 11.7 with an Intel 12700H CPU.

**Comment by JRPan** [comment]
I'm having a similar issue with a 13th gen Intel CPU. I think the Mesa driver needs to be updated to support the latest Intel GPUs. Can someone confirm that? And is there any plan for that? 

Thanks!

**Comment by mohammadrezasaed** [comment]
The version of Mesa in Vulkan-Sim doesn't support 13th gen Intel CPU. We will release a version of Vulkan-Sim that doesn't need intel integrated GPUs.

**Comment by tommychouyc** [comment]
We've just released Vulkan-Sim 2.0.0 and the corresponding mesa repo which removes the Intel driver dependency and can now be run directly on any CPU.
